cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java

```diff
@@ -90,6 +93,7 @@
   private boolean suspended;
   private Lock lock;
   private Condition condition;
+  private final LoggingContext loggingContext;
```

Comment:
Usually I prefer having all `final` fields together and before any non-final field.